src/sharedData/components/SharedDataUpload/ShareDataLinks.js

```diff
@@ -0,0 +1,315 @@
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import _ from 'lodash/fp';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import * as yup from 'yup';
+
+import AddIcon from '@mui/icons-material/Add';
+import DeleteIcon from '@mui/icons-material/Delete';
+import {
+  Alert,
+  Button,
+  Divider,
+  IconButton,
+  LinearProgress,
+  Stack,
+  Typography,
+} from '@mui/material';
+
+import ExternalLink from 'common/components/ExternalLink';
+import { transformURL } from 'common/utils';
+import Form from 'forms/components/Form';
+import { FormContextProvider, useFormGetContext } from 'forms/formContext';
+
+import LinkIcon from '../LinkIcon';
+import SuccessContainer from './SuccessContainer';
+import { groupByStatus } from './utils';
+
+export const VALIDATION_SCHEMA = yup
+  .object({
+    url: yup.string().trim().ensure().transform(transformURL).url().required(),
+    name: yup.string().trim().required(),
+    description: yup.string().maxCustom(200).trim(),
```

Comment:
question: what is 200?
issue: put 200 in a constant.

Reply:
@paulschreiber max characters, is is used under string(), I am not sure why this is using maxCustom instead of max directly, I think I did this before knowing how yup works, I will check this